.gitignore

```diff
@@ -8,6 +8,7 @@ pnpm-debug.log*
 lerna-debug.log*
 
 node_modules
+package-lock.json
```

Comment:
I missed this, this is a no go, do not add package locks to git ignore
See: https://stackoverflow.com/a/44210813

Reply:
Thank you for the link. I had no idea this was a bad practice. I added `package-lock.json` back in 5c1937a943831befa3feef898ee8e9e93a2dc51a